### PR TITLE
Add commands to activate and deactivate endpoints, message processors and proxy services in MI instances

### PR DESCRIPTION
**Pull request by SanojPunchihewa**
## Purpose
Update MI CLI tool and merge with APICTL
wso2/micro-integrator#1999

## Goals
- Add a new activate command under mi alias to activate endpoints, message processors, and proxy services in MI instances
- Add a new deactivate command under mi alias to deactivate endpoints, message processors, and proxy services in MI instances

## Approach
- Created a new package called activate inside the mi package
- Add the necessary sub-commands(endpoints, message processors, and proxy services) inside that get package
- Created a new package called deactivate inside the mi package
- Add the necessary sub-commands(endpoints, message processors, and proxy services) inside that get package

## User stories
- To activate an endpoint
`apictl mi activate endpoint [endpoint-name]`
- To activate a message processor
`apictl mi activate message-processor [message-processor-name]`
- To activate a proxy service
`apictl mi activate proxy-service [proxy-name]`

- To deactivate an endpoint
`apictl mi deactivate endpoint [endpoint-name]`
- To deactivate a message processor
`apictl mi deactivate message-processor [message-processor-name]`
- To deactivate a proxy service
`apictl mi deactivate proxy-service [proxy-name]`

## Test environment
Ubuntu 20.04.1 LTS
Go version go1.15 linux/amd64

### CODE REVIEW

File: import-export-cli/mi/impl/common.go [hunk 1]
```diff
@@ -237,3 +238,12 @@ func createErrorWithResponseBody(resp string, err error) error {
 	}
 	return err
 }
+
+func updateArtifactState(url, artifactName, state, env string) (string, error) {
+	body := dedent.Dedent(`{
+		"name": "` + artifactName + `",
+		"status": "` + state + `"
+	}`)
```
Comment (uvindra) [inline import-export-cli/mi/impl/common.go:246]: Any reason for not using a struct instead? We can avoid needing to depend on the third party package `dedent`

File: import-export-cli/cmd/mi/activate/activate.go [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+var ActivateCmd = &cobra.Command{
+	Use:     activateCmdLiteral,
+	Short:   activateCmdShortDesc,
+	Long:    activateCmdLongDesc,
+	Example: activateCmdExamples,
+	Run: func(cmd *cobra.Command, args []string) {
+		utils.Logln(utils.LogPrefixInfo + activateCmdLiteral + " called")
+		cmd.Help()
+	},
+}
```
Comment (wasuradananjith) [inline import-export-cli/cmd/mi/activate/activate.go:43]: Shall we add a new line to the end of the file? (It is a good practice to do [1]. Better to do the change where all applicable)
[1] https://www.quora.com/Why-is-it-a-common-practice-to-put-a-new-line-at-the-end-of-a-source-code-file

Comment (SanojPunchihewa) [reply import-export-cli/cmd/mi/activate/activate.go:43]: I have already added the new line and it seems git is not showing it. If there is no new line git will show the :no_entry: at the end of the file AFAIK.

Comment (wasuradananjith) [reply import-export-cli/cmd/mi/activate/activate.go:43]: No worries then. You can ignore this comment. Thanks!

File: import-export-cli/mi/impl/common.go (hunk 1 above)
Comment (SanojPunchihewa) [reply import-export-cli/mi/impl/common.go]: Fixed with a0db0eb. Did some refactoring to `InvokePOSTRequest` to use interfaces so we can pass strings, structs, maps or byte arrays directly as the body

**Review by uvindra — APPROVED** [review]

**Review by wasuradananjith — APPROVED** [review]

**Pull request merged**